opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -389,12 +388,11 @@ message SummaryValue {
 
   // The total number of recorded values since start_time. Optional since
   // some systems don't expose this.
-  google.protobuf.Int64Value count = 3;
+  uint64 count = 3;
```

Comment:
fixed64?

Reply:
Given that most values are small that may increase wire size (not a big deal). I do not think we will gain anything from fixed64 in this case.
I would say it doesn't matter, can change if you have an opinion about it.

Reply:
This is the total number of samples observed from the beginning of the process. This will easily grow.

Reply:
I still want to support sampled metrics, in which case the count could be approximate (I'd suggest `double`). Maybe it would be appropriate to use a separate field to represent approximate counts, though.

Reply:
I think we can use `double` universally. It is going to be a negligible performance hit to convert from int if needed.

But, I am not sure I understand the need for fractional count. How do you sample a metric or more precisely why would you need to lose the information about the real count (I am guessing by multiplying `count` and `sum` by some sampling coefficient)?

Reply:
I created a separate issue to discuss the need for fractional counts: https://github.com/open-telemetry/opentelemetry-proto/issues/59
Let's merge this one as is for now.

Reply:
@bogdandrutu done.

Reply:
Wait, wrong place :-(

Reply:
@bogdandrutu OK, it is fixed64 now.

Reply:
I don't see why fixed64 makes sense here.  We're talking about a Summary, which implies it is infrequently reported, so probably CPU usage is not a concern.  An int64 is a smaller encoding for values up to 2^49 or so.

Anyway it's not a big deal.

Reply:
I believe the difference is negligible. int64 is slightly slower to encode/decode and slightly smaller uncompressed but (IIRC) often compresses worse than fixed64 by zlib (I didn't dig into why - could entire because of my test data). The combined difference is below my threshold for optimizations (and below my measurement error for CPU time).
I am fine with either approach.

Reply:
That's a good point--about compression. Thanks.